python/cuml/fil/fil.pyx

```diff
@@ -236,6 +236,25 @@ cdef class ForestInference_impl():
                       &treelite_params)
         return self
 
+    def load_from_randomforest(self,
+                               model_handle,
+                               bool output_class,
+                               str algo,
+                               float threshold):
+        cdef treelite_params_t treelite_params
```

Comment:
Add some blank lines in this method (probably before the line 248 or so), just a nitpick but this blob of code is just very hard to read